common/src/main/java/com/wynntils/features/user/players/PlayerGhostTransparencyFeature.java

```diff
@@ -30,4 +34,13 @@ public void onTranslucentCheck(LivingEntityRenderTranslucentCheckEvent e) {
             e.setTranslucence(playerGhostTranslucenceLevel);
         }
     }
+
+    @SubscribeEvent
+    public void onGhostArmorRender(PlayerGhostArmorRenderEvent e) {
+        if (!WynnUtils.onWorld()) return;
```

Comment:
Events are only posted if this condition is met, remove this

Reply:
Actually, events are only posted if `onServer`. If we want to disable this functionality on the hub, this check is needed. (I don't know if this is the case here, though)

Reply:
Doesn't matter here